lib/tests/streamlit/hashing_test.py

```diff
@@ -710,3 +710,21 @@ def g(y):
         self.assertEqual(
             get_hash(f, hash_funcs=hash_funcs), get_hash(g, hash_funcs=hash_funcs)
         )
+
+    def test_ufunc(self):
+        """Test code that references numpy ufuncs."""
+
+        def f(a, b):
+            return np.logical_and(a, b)
+
+        def g(a, b):
+            return np.logical_and(a, b)
+
+        def h(a, b):
+            return np.remainder(a, b)
+
+        self.assertNotEqual(get_hash(np.remainder), get_hash(np.logical_and))
+        self.assertEqual(get_hash(np.logical_and), get_hash(np.logical_and))
```

Comment:
This line can be removed as we typically don't do this in the other hashing tests